tests/integration_tests/test_time_series_integration.py

```diff
@@ -0,0 +1,186 @@
+"""Authors: Cody Baker and Ben Dichter."""
+import numpy as np
+from unittest import TestCase
+from shutil import rmtree
+from tempfile import mkdtemp
+from uuid import uuid4
+from datetime import datetime
+from pathlib import Path
+
+import pynwb
+
+from nwbinspector import (
+    check_dataset_compression,
+    check_regular_timestamps,
+    check_data_orientation,
+    check_timestamps_match_first_dimension,
+)
+
+
+class PreWriteIntegrationTestTimeSeriesChecks(TestCase):
+    def setUp(self):
+        self.nwbfile = pynwb.NWBFile(
+            session_description="Testing inspector.",
+            identifier=str(uuid4()),
+            session_start_time=datetime.now().astimezone(),
+        )
+
+    def test_check_regular_timestamps(self):
+        time_series = pynwb.TimeSeries(
+            name="test_time_series",
+            unit="test_units",
+            data=np.zeros(shape=3),
+            timestamps=[1.2, 3.2, 5.2],
+        )
+        self.nwbfile.add_acquisition(time_series)
+        self.assertEqual(
+            first=check_regular_timestamps(
+                time_series=self.nwbfile.acquisition["test_time_series"]
+            ),
+            second=(
+                "TimeSeries appears to have a constant sampling rate. "
+                f"Consider specifying starting_time={time_series.timestamps[0]} "
+                f"and rate={time_series.timestamps[1] - time_series.timestamps[0]} instead of timestamps."
+            ),
+        )
```

Comment:
These are close to how small I'd like unit tests to be, except I don't think this requires the creation of an nwbfile in `setUp`

Reply:
These are integration tests, hence the `nwbfile`.

See https://github.com/catalystneuro/nwbinspector/blob/add_testing_and_refactor/tests/unit_tests/test_time_series_unit.py#L14-L29 for the unit test for this function